direct-messages/README.md

```diff
@@ -10,24 +10,74 @@ because the same message can simultaneously:
 - give the new person the info needed to access the group
 - reveal to the group that this action happened
 
-## Mapping `feed_id` to a `recp_key`
+## Computing the `recp_key`
 
 We define a shared key that the sender + recipient can both derive:
 
 ```js
-function computeDirectMessageKey (my_secret, your_public) {
+function computeDirectMessageKey (my_dh_secret, my_dh_public, your_dh_public, my_feed_tfk, your_feed_tfk) {
   var hash = 'SHA256'
-  var salt = SHA256("envelope-direct-messsage-shared-key-extract-salt")
-  var input_keying_material = scalarmult(my_secret, your_public)
-
-  return hkdf.Extract(hash, salt, input_keying_material)
+  var length = 32
+  var salt = SHA256("envelope-dm-v1-extract-salt")
+  var info = slp_encode(
+      ["envelope-ssb-dm-v1/key"]
+      .concat(sort([
+		tfk_encode(my_dh_public) || my_feed_tfk,
+		tfk_encode(your_dh_public) || your_feed_tfk
+  ])))
```

Comment:
you've added e.g. `tfk_encode(my_dh_public)` to this info, which is different that what we drafted on our call.

any particular reason ? 

reference:
```js
function computeDirectMessageKey (my_cv_secret, your_cv_public, my_sign_public, your_sign_public) {
  // all args are Buffers (cv are raw keys, signing keys are TFK)
  
  var hash = 'SHA256'
  var length = 256/8 // 256 bit key
  var input_keying_material = scalarmult(my_cv_secret, your_cv_public)
  var info = slp_encode(["dm-key-v1"].concat(sort([my_sign_public, your_sign_public])))
  var salt = "envelope-id-based-dm-converted-ed25519"
  // see 'private-group-spec/direct-messages/constants.json'
  
  return hkdf(input_keying_material, length, {salt, info, hash})
}
```

Reply:
Oh I see some justification below now